src/profile-logic/call-tree.js

```diff
@@ -170,6 +170,18 @@ export class CallTree {
     };
   }
 
+  getTimingDisplayData(callNodeIndex: IndexIntoCallNodeTable) {
+    const totalTime = this._callNodeTimes.totalTime[callNodeIndex];
+    const selfTime = this._callNodeTimes.selfTime[callNodeIndex];
+    const formatNumber = this._isIntegerInterval
+      ? _formatIntegerNumber
+      : _formatDecimalNumber;
+    return {
+      totalTime: `${formatNumber(totalTime)}`,
+      selfTime: selfTime === 0 ? '—' : `${formatNumber(selfTime)}`,
+    };
+  }
```

Comment:
Can you please call this method from `getDisplayData` below?

This could be as simple as writing:
```javascript
displayData = {
  ...this.getTimingDisplayData(callNodeIndex),
  foo,
  bar
};
```
This syntax is possible if we enable https://babeljs.io/docs/plugins/transform-object-rest-spread/, which I'd like to do today to make it easier.

This also makes it possible to remove the definition of `formatNumber` in that function and simplifies it a bit.

Reply:
https://github.com/devtools-html/perf.html/pull/868 landed, you should be able to rebase and profit :)

Reply:
Awesome! I've rebased and added the call :-)